### PR TITLE
Add WatsonX metrics test

### PR DESCRIPTION
**Pull request by bdattoma**
This PR is adding a test for checking model metrics for WatsonX/Kserve. 

The PR checks:
1. metrics with `tgi_`, `caikit_` and `istio_` are present in the Openshift metrics
2. user can get `# of inference requests over defined time period` , `# of successful inference requests over defined time period` and `CPU Utilization`

This is a base for the final test which should check that all the expected metrics are present (_waiting for a final list_) and will check also more derived metrics (_waiting for a final list and their formulas)

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 1]
```diff
@@ -671,12 +713,21 @@ Deploy Caikit Serving Runtime
     ...    oc apply -f ${CAIKIT_FILEPATH} -n ${namespace}
 
 Set Project And Runtime
-    [Arguments]    ${namespace}
+    [Documentation]    Creates the DS Project (if not exists), creates the data connection for the models,
+    ...                creates caikit runtime. This can be used as test setup
+    [Arguments]    ${namespace}    ${enable_metrics}=${FALSE}
+    ${test_var}=    Set Variable    hello there!
```
Comment (FedeAlonso) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:719]: Is this variable used? Is the value right?

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 2]
```diff
@@ -429,13 +431,53 @@ Verify User Can Serve And Query Flan-t5 Large
     [Teardown]    Clean Up Test Project    test_ns=${test_namespace}
     ...    isvc_names=${models_names}
 
+Verify User Can Access Model Metrics From UWM
+    [Documentation]    Verifies that model metrics are available for users in the
+    ...                OpenShift monitoring system (UserWorkloadMonitoring)
+    ...                PARTIALLY DONE: it is checking number of requests, number of successful requests
+    ...                and model pod cpu usage. Waiting for a complete list of expected metrics and
+    ...                derived metrics.
+    [Tags]    ODS-XYZ    WatsonX
```
Comment (FedeAlonso) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:440]: ODS-XYZ means that the test is not defined in Polarion?

**Review by FedeAlonso — CHANGES_REQUESTED** [review]

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot (hunk 1 above)
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot]: no it was used for debug reasons and removed

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot (hunk 2 above)
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot]: correct, i've replace it already with a real value

**Review by FedeAlonso — DISMISSED** [review]
LGTM

**Comment by bdattoma** [comment]
verified at `rhods-ci-pr-test/2064` SUCCESS

**Comment by tarukumar** [comment]
@bdattoma why don't you enable metric in Suite setup. I don't see any harm in that those are cluster wide configuration and having it enable by default will be good idea.

**Comment by bdattoma** [comment]
> @bdattoma why don't you enable metric in Suite setup. I don't see any harm in that those are cluster wide configuration and having it enable by default will be good idea.

I thought to have scenarios running without metrics enabled, since users won't be forced to enable them. But yeah I can add it to suite setup, do you mind if we move it there in another PR? this one has been already tested

**Review by FedeAlonso — APPROVED** [review]
LGTM

**Review by tarukumar — APPROVED** [review]

**Pull request merged**